CRM/MembershipExtras/Job/OfflineAutoRenewal/PaymentPlan.php

```diff
@@ -230,6 +231,23 @@ public function run() {
    */
   abstract public function renew();
 
+  /**
+   * Creates new periods for memberships that were renewed.
+   */
+  private function createMembershipPeriods() {
+    $recurringLineItems = $this->getNewPaymentPlanActiveLineItems();
+
+    foreach ($recurringLineItems as $lineItem) {
+      $priceFieldValue = !empty($lineItem['price_field_value_id']) ? $this->getPriceFieldValue($lineItem['price_field_value_id']) : [];
+      if (!$this->isMembershipLineItem($lineItem, $priceFieldValue)) {
+        continue;
+      }
+
+      $existingMembershipID = $this->getExistingMembershipForLineItem($lineItem, $priceFieldValue);
```

Comment:
I would rename this method to **getExistingMembershipIdForLineItem** to indicate that it return an id